Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -2074,3 +2139,10 @@ PhysicalParticleContainer::buildIonizationMask (const amrex::MFIter& mfi, const
         }
     );
 }
+
+//This function return true if the PhysicalParticleContainer contains electrons
+//or positrons, false otherwise
+bool
+PhysicalParticleContainer::AmIALepton(){
+    return (this-> mass == PhysConst::m_e);
```

Comment:
Another reason for allowing the user to specify the physical type of particles (electron, ion, etc.)... That's still on my to-do list! (note that it wouldn't impact this PR).